src/components/organisms/Slider/CardAuthor.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import { UserInfo } from '../../molecules/UserInfo/UserInfo'
+import { Avatar } from '../../atoms/Avatar/Avatar'
+
+export interface CardAuthorProps {
```

Comment:
Please extend the `DefaultComponentProps` type to handle `className` and `style` like mentioned in the CONTRIBUTING file

Reply:
these components were removed since they were part of business logic. sorry for the confusion.
But yes, I'll extend DefaultComponentProps for all the future components to ensure style and className props.

Reply:
Removed? but they are still there what do you mean?

Reply:
sorry, forgot to push the changes after removing!

Reply:
No problem please don't forget to remove the branch 😉